### ISSUE TITLE
Issue in PortChannel - Removing given vlan members from port channel

### ISSUE THREAD

**Issue by rahul-1475** [STORDIS/orca_nw_lib#76]
Adding addtional functionaltity to removing given vlan members from port channel.

- [x] code to deleting given access / trunk vlans.
- [x] implementing in test cases to check it

**Comment by rahul-1475** [comment]
orca_backend pr: https://github.com/STORDIS/orca_backend/pull/48
orca_nw_lib pr: https://github.com/STORDIS/orca_nw_lib/pull/77

**Comment by rahul-1475** [comment]
Once the pr is merged and closed @vivek-metricdust for this respective frondend task should be taken.
ie., Issue in PortChannel - Enabling remove button to delete vlan members in port channel
[#137](https://github.com/STORDIS/orca_ui/issues/137)

**Comment by kamalkrbh** [comment]
@vivek-metricdust , For deletion of mem vlans from a port channel the changes has to be adapted in UI in a new task.
